core/src/banking_stage.rs

```diff
@@ -372,20 +377,24 @@ impl LikeClusterInfo for Arc<ClusterInfo> {
 impl BankingStage {
     #[allow(clippy::too_many_arguments)]
     pub fn new_num_threads(
+        cluster_info: Arc<ClusterInfo>,
         block_production_method: BlockProductionMethod,
         poh_recorder: Arc<RwLock<PohRecorder>>,
         transaction_recorder: TransactionRecorder,
         non_vote_receiver: BankingPacketReceiver,
         tpu_vote_receiver: BankingPacketReceiver,
         gossip_vote_receiver: BankingPacketReceiver,
+        verified_bundle_receiver: Receiver<VerifiedPacketBundle>,
         num_workers: NonZeroUsize,
         scheduler_config: SchedulerConfig,
         transaction_status_sender: Option<TransactionStatusSender>,
         replay_vote_sender: ReplayVoteSender,
         log_messages_bytes_limit: Option<usize>,
         bank_forks: Arc<RwLock<BankForks>>,
+        block_builder_fee_info: &Arc<Mutex<BlockBuilderFeeInfo>>,
         prioritization_fee_cache: Arc<PrioritizationFeeCache>,
         blacklisted_accounts: HashSet<Pubkey>,
+        tip_manager: TipManager,
         bundle_account_locker: BundleAccountLocker,
         batch_interval: Duration,
```

Comment:
Why this random additions? would be much easier to maintain conflicts if args are added at the end